Header header
string[] classes
float64[] depths
float64[] centers_x
float64[] centers_y